eng/scripts/Create-APIView.ps1

```diff
@@ -86,6 +86,32 @@ function Get-ResourceProviderFromReadMePath {
     return $null
 }
 
+function Get-ImpactedTypespecProjects {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$TypeSpecFile
+    )
+    $filePathParts = $TypeSpecFile.split([IO.Path]::DirectorySeparatorChar)
+    while ($filePathParts.Length -and !$configFilesInTypeSpecProjects) {
+      $filePathParts = $filePathParts | Select-Object -SkipLast 1
+      $typeSpecProjectBaseDirectory = $filePathParts -join [IO.Path]::DirectorySeparatorChar
+      $configFilesInTypeSpecProjects = Get-ChildItem -Path $typeSpecProjectBaseDirectory -File "tspconfig.yaml"
+    }
+    
+    if ($configFilesInTypeSpecProjects) {
+      foreach($configFilesInTypeSpecProject in $configFilesInTypeSpecProjects) {
+        $entryPointFile = Get-ChildItem -Path $($configFilesInTypeSpecProject.Directory.FullName) -File "main.tsp" 
```

Comment:
Why main.tsp? What about client.tsp?

Reply:
we wanted to determine which TypeSpec projects were affected and the old apiview one considers the directory a project by having both main.tsp and tspconfig.yaml, after looking at the tsv folder structure rule seems like right now a directory will be consider a project if it contains tspconfig.yaml and (either main.tsp or client.tsp). I've made the change and thanks for the heads up on this Wes~